<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>属性：</string>
   </property>
   <widget class="QCheckBox" name="chkWrite">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>写</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRead">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>读</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>40</y>
      <width>73</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>目录</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkHide">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>隐藏</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkLink">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>符合连接</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkExec">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnGetInfo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取文件信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>451</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labFileName">
      <property name="text">
       <string>文件名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leFileName"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnFileName">
        <property name="text">
         <string>文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labSize">
      <property name="text">
       <string>大小：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leSize"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labCreateTime">
      <property name="text">
       <string>创建时间：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leCreateTime"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labModifyTime">
      <property name="text">
       <string>最后修改时间：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leModifyTime"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labVisitTime">
      <property name="text">
       <string>最后访问时间：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="leVisitTime"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
